gufe/settings/models.py

```diff
@@ -0,0 +1,174 @@
+    scale12: float
+    scale13: float
+    scale14: float
+    scale15: float
+
+
+class VdWSettings(SettingsBaseModel):
+    """
+    Settings for van der Waals force
+    """
+
+    combining_rules: Literal["Lorentz-Berthelot"]
+    potential: Literal["Lennard-Jones-12-6"]
+    scale: vdWScale
+    long_range_dispersion: Literal["isotropic"]
+    cutoff: float  # angstrom
+    switch_width: float  # angstrom
+    method: Literal["cutoff"]
+
+
+class ElectrostaticSettings(SettingsBaseModel):
+    """
+    Settings for electrostatics
+    """
+
+    # Tricky since this allows functions https://openforcefield.github.io/standards/standards/smirnoff/#electrostatics
+    periodic_potential: Union[Literal["Ewald3D-ConductingBoundary"], str]
+    nonperiodic_potential: Union[Literal["Coulomb"], str]
+    exception_potential: Union[Literal["Coulomb"], str]
+    scale: ElectrostaticScale
+    cutoff: Union[float, None]
+    switch_width: Union[float, None]
+    solvent_dielectric: Union[float, None]
+
+
+class BondSettings(SettingsBaseModel):
+    """
+    Settings for harmonic bonds
+    """
+
+    # U(r) = (k/2)*(r-length)^2
+    potential: Literal["harmonic"]
+    # Might support other methods?
+    fractional_bondorder_method: Literal["AM1-Wiberg"]
+    fractional_bondorder_interpolation: Literal["linear"]
+
+
+class AngleSettings(SettingsBaseModel):
+    """
+    Settings for harmonic angles
+    """
+
+    # U(r) = (k/2)*(theta-angle)^2
+    potential: Literal["harmonic"]
+
+
+class ProperTorsionSettings(SettingsBaseModel):
+    """
+    Settings for proper torsions
+    """
+
+    # U = \sum_{i=1}^N k_i * (1 + cos(periodicity_i * phi - phase_i))
+    potential: Literal["k*(1+cos(periodicity*theta-phase))"]
+    default_idivf: Union[Literal["auto"], int] = "auto"
+    # Might support other methods?
+    fractional_bondorder_method: Literal["AM1-Wiberg"]
+    fractional_bondorder_interpolation: Literal["linear"]
+
+
+class ImproperTorsionSettings(SettingsBaseModel):
+    """
+    Settings for improper torsions
+    """
+
+    # U = \sum_{i=1}^N k_i * (1 + cos(periodicity_i * phi - phase_i))
+    potential: Literal["k*(1+cos(periodicity*theta-phase))"]
+    default_idivf: Literal["auto"]
+
+
+class GBSASettings(SettingsBaseModel):
+    """ "
+    Settings for Generalized-Born surface area (GBSA) implicit solvent parameters
+    """
+
+    gb_model: Literal["HCT", "OBC1", "OBC2"]
+    solvent_dielectric: float
+    solute_dielectric: float
+
+
+class ForcefieldSettings(SettingsBaseModel):
+    """
+    Settings for the forcefield
+    """
+
+    # Metadata
+    date: Union[date, None]
+    author: Union[str, None]
+
+    # These should also allow None
+    vdW: VdWSettings
+    electrostatics: ElectrostaticSettings
+    bonds: BondSettings
+    angles: AngleSettings
+    proper_torsions: ProperTorsionSettings
+    improper_torsions: ImproperTorsionSettings
+    gbsa: GBSASettings
+
+    # Misc
+    aromaticity_model: Literal["OEAroModel_MDL"]
```

Comment:
Not sure we need this in the FF definition?  Isn't aromaticity already defined in the chemical models (elsewhere)

Reply:
It matters in how rdkit will interpret things, so if the FF was parametrized with that assumption it could matter. This is less important if we don't care about fully supporting openeye https://docs.eyesopen.com/toolkits/python/oechemtk/aromaticity.html#aromaticity-models-in-oechem-tk

Reply:
My thought was that if we're providing a fully interpreted `openff.toolkit.Molecule` is there any more interpretation of the aromaticity happening?  I'll ask over on the ff slack

Reply:
Ok from asking around, this isn't actually used, so we can remove it